packages/base/src/Boot.ts

```diff
@@ -40,6 +41,13 @@ const boot = async (): Promise<void> => {
 			return;
 		}
 
+		attachThemeRegistered(() => {
+			if (!booted) {
+				return;
+			}
+			applyTheme(getTheme());
```

Comment:
may be we should check if the theme registered is the one that will be applied `getTheme`. there is no need to call `applyTheme` if the registered theme won't be applied.